ocs_ci/helpers/helpers.py

```diff
@@ -2972,3 +2973,72 @@ def run_cmd_verify_cli_output(
         if expected_output not in out:
             return False
     return True
+
+
+def check_pvs_present_for_ocs_expansion(sc=constants.LOCALSTORAGE_SC):
```

Comment:
I think we can move this function to the file https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/pv.py since it's related to pv functions. Also, in this file, we have helper functions like `get_pv_status` and `get_pv_name`.

Reply:
Done